index.html

```diff
@@ -28,24 +28,192 @@ <h2 class="highligh-title">I'm a software developer</h2>
+              </button>
+            </div>
+          </div>
+          <div class="grid-item-1">
+            <div class="single-card">
+              <h3>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h3>
+              <ul class="skills-list">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+              <button class="see-project-button" type="button">
+                See Project
+              </button>
+            </div>
+          </div>
+          <div class="grid-item-1">
+            <div class="single-card">
+              <h3>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h3>
+              <ul class="skills-list">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+              <button class="see-project-button" type="button">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about-me-detail">
+        <div class="about-me-head">
+          <h2>About me</h2>
+          <p>
+            I can help you build a product, feature or website Look through some
+            of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <button class="aboutme-button" type="button">Get My Resume</button>
+        </div>
+        <div class="grid-container-2">
+          <div class="grid-item-2">
+            <div class="single-card-2">
+              <img src="images/Ellipse-lang.png" alt="lang" />
+              <h3>Language</h3>
+              <ul class="skills-list-2">
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>html</li>
+                <li>css</li>
+              </ul>
+            </div>
+          </div>
+          <div class="grid-item-2">
+            <div class="single-card-2">
+              <img src="images/Ellipse-frame.png" alt="lang" />
+              <h3>Frameworks</h3>
+              <ul class="skills-list-2">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Selenium</li>
+              </ul>
+            </div>
+          </div>
+          <div class="grid-item-2">
+            <div class="single-card-2">
+              <img src="images/Ellipse-skill.png" alt="lang" />
+              <h3>Skills</h3>
+              <ul class="skills-list-2">
+                <li>Codekit</li>
+                <li>GitHub</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+              </ul>
+            </div>
+          </div>
```

Comment:
@chaw-bot Thank you for the review. I only have a question on this part because as you can see I have put the categories under `ul` and each of the 'languages', 'frameworks', and 'Skills' in `li`.
Just like the requirement:
- Please use ul and li for categories in "Work" section. Html, Css, and Javascript should be list items.
- Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.